src/renderers/WebGLRenderer.js

```diff
@@ -2251,4 +2251,28 @@ function WebGLRenderer( parameters = {} ) {
 
 }
 
+Object.defineProperties( WebGLRenderer.prototype, {
+
+	// @deprecated since r147
+
+	physicallyCorrectLights: {
+
+		get: function () {
+
+			console.warn( 'THREE.WebGLRenderer: the property .physicallyCorrectLights has been removed. Set renderer.useLegacyLights instead.' );
+			return undefined;
```

Comment:
Why not return `! this.useLegacyLights`?

Reply:
I was expecting that question. :-)

I'll change it.